tests/metrics/density/docker_cpu_usage.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#  Description of the test:
+#  This test checks the CPU usage of several idle containers and
+#  containers that are working running shuf.
+#  The measure is take after the workload has become stable.
+#  By default this tests run 100 containers and waits 2m to stabilize the
+#  workloads.
+
+set -e
+
+if [ $# -ne 2 ]; then
+  echo -e "Usage: $0 <times to run> <wait time>" && exit 1
+fi
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "$SCRIPT_PATH/../common/test.common"
+
+TEST_NAME="Qemu CPU % usage"
+IMAGE='ubuntu'
+RUNTIME='cor'
+TIMES=$1
+WAIT_TIME=$2
+
+function get_idle_cpu_usage(){
+  testArgs="rootfs=$IMAGE idle_mode"
+  testResultFile="$RESULT_DIR/cpu-usage-idle"
+  backup_old_file "$testResultFile"
+  write_csv_header "$testResultFile"
+  if [ $RUNTIME == 'cor' ]; then
+    for i in $(seq 1 "$TIMES"); do
+      $DOCKER_EXE run -tid $IMAGE bash
+    done
+    sleep "$WAIT_TIME"
+    qemu_pids=$(pgrep qemu-lite)
```

Comment:
Rather than hard-coding `qemu-lite`, this should be `@QEMU_PATH@` (and then the usual rename to `tests/metrics/density/docker_cpu_usage.sh.in`, and adding `tests/metrics/density/docker_cpu_usage.sh` to `GENERATED_FILES` and adding to `tests/metrics/density/docker_cpu_usage.sh.in` to `EXTRA_DIST` in `Makefile.am`.